pandas/tests/extension/base/setitem.py

```diff
@@ -93,6 +93,90 @@ def test_setitem_iloc_scalar_multiple_homogoneous(self, data):
         df.iloc[10, 1] = data[1]
         assert df.loc[10, "B"] == data[1]
 
+    @pytest.mark.parametrize(
```

Comment:
think about adding a new class here; these are specifically for setitem_mask (could be a followon)

Reply:
> think about adding a new class here; these are specifically for setitem_mask (could be a followon)

In general, changing names is annoying for downstream consumers of those tests (this are the base extension tests), so not worth here I think

Reply:
ok, I don't buy your argument, these are experimental features where we actually to expand test coverage. its going to happen (but ok for now)